pybamm/models/full_battery_models/lithium_ion/base_lithium_ion_model.py

```diff
@@ -254,6 +281,40 @@ def set_open_circuit_potential_submodel(self):
                     self.param, domain, reaction, self.options, phase
                 )
 
+    def set_pe_degradation_submodel(self):
+        # specify the degradation domain
+        domain = "positive"
+
+        # replace the fickian particle submodel
+        # make sure set_pe_degradation_submodel comes after
```

Comment:
there is already code that loops over the submodels in a way that the order doesn't matter. does it fail in this case? if so, it would be good to see if we can fix that part of the code rather than having a special case.